.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.10",
+	"features": {
+		"ghcr.io/devcontainers/features/node:1": {}
+	},
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [8000],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "pip3 install --user -r requirements.txt && npm install && mkdir -p data && python3 manage.py migrate && playwright install chromium && playwright install-deps",
```

Comment:
`playwright install-deps` failed for me:
```
E: Package 'ttf-ubuntu-font-family' has no installation candidate
E: Unable to locate package libenchant1c2a
E: Unable to locate package libicu66
E: Package 'libjpeg-turbo8' has no installation candidate
Failed to install browser dependencies
Error: Installation process exited with code: 100
[120273 ms] postCreateCommand failed with exit code 1. Skipping any further user-provided commands.
```
Let's remove the playwright installation for now, I'll figure out how to run the playwright tests separately at some point.

Reply:
Interesting, I wonder why the command works for me and not for you, that shouldn't happen. If it becomes a problem we can use a base Dockerfile for the Dev Container rather then the Microsoft Python Container, that way it can be pre-setup with all dependencies that are required.

For now I have removed the playwright commands. Let me know if there are any other changes required,